readme.md

```diff
@@ -106,18 +106,28 @@ When merging into a newer MPS version, follow these steps.
 The example assumes we merge `mps2021.1` into `mps2021.2`.
 
 1. Open the _target_ MPS version (2021.2) on the _target_ branch (`mps2021.2`).
-2. Merge the _source_ branch (`mps2021.1`) into your _current_ branch (`mps2021.2`).
-3. Double-check `gradle.properties` still contains the proper entries for:
+2. Checkout a new branch off of the _target_ branch (`mps2021.2`), named `mps2021.2-migration` (or a name that's as least as good).
+3. Merge the _source_ branch (`mps2021.1`) into your _current_ branch (`mps2021.2-migration`).
+4. Double-check `gradle.properties` still contains the proper entries for:
    * `mpsVersionSuffix` should be the _target_ MPS version (`2021.2`)
    * `mpsVersion` full _target_ MPS version (`2021.2.6`)
    * `mpsExtensionsVersion` latest version of [MPS-extensions](https://jetbrains.github.io/MPS-extensions/) for the _target_ MPS version (`2021.2.2631.1360a64`)
-4. Run the Migration Assistant.
-5. Run all tests.
-6. Update the build model.
-7. Repeat steps 3-6 for the test projects, residing in `test-project/` and `test-project-externalLib/`.
-  **Note** Before opening these projects in MPS:
-    * Make sure to rename their build model files (the `.mps` files in the `models/` directory under the one solution under `solutions/` of either project) to not have the `-ignore` suffix.
-    * Make sure to put those suffixes back in place before committing.
+5. Run the Migration Assistant.
+6. Update the (two) build models, triggering the “Reload Modules From Disk” intention when and where needed.
+7. Check the entire project using “Check Project”.
+  Investigate and fix any errors outside the module named “`xx_broken`”.
+8. Rebuild the (entire) project.
+9. Run all tests, in the standard way from within MPS using “Run 'All Tests in Project'”.
+  **Note** that not all tests will succeed when run within MPS: some that succeed when run in CI (i.e.: in the GitHub Action), and some that are currently broken.
+  The former can be recognized by them throwing a `jetbrains.mps.baseLanguage.unitTest.execution.server.InProcessExecutionFilter$TestSetNotToBeExecutedInProcessException`.
```

Comment:
Nope, they can be recognized by a comment on the test.
This is because some dependencies are different when running in headless vs. UI. I gave up debugging why. The expectations match what's happening in headless, giving errors in UI.

Reply:
Are those comments there?

Reply:
yes, e.g. http://127.0.0.1:63320/node?ref=r%3A43c660c3-adeb-4a6a-893f-396c007e80f0%28io.lionweb.mps.converter.test.languagedependsonfinder%40tests%29%2F1313442573159668044

Reply:
But is e.g. this one not erroring out for you, then?: http://127.0.0.1:63320/node?ref=r%3Aec257433-77a2-4e03-b9c4-d2c4dc38da22%28io.lionweb.mps.converter.test.concept%40tests%29%2F5938343903671319145